packages/tools/eslint-config-pie/__tests__/eslint-rules.test.js

```diff
@@ -50,7 +50,7 @@ describe('ESLint', () => {
 
             expect(results.length).toBe(1);
             expect(results[0].messages).toMatchSnapshot();
-        });
+        }, 30000); // Add 30 second timeout for CI environments
```

Comment:
This might not be needed given the timeout in package.json - if possible, I'll raise a separate PR to remove this given this PR affects all components, which has meant Percy flakiness / timeouts are quite frequent